trace/requirements.txt

```diff
@@ -1 +1,10 @@
-google-gax>=0.15.7, <0.16dev
+Django==1.11.5
```

Comment:
Can you move this to `requirements-test.txt`, and leave the existing `requirements.txt` as is?  We want to separate the dependencies a customer needs to install this (just google-gax), from the dependencies needed to test this (everything).

Reply:
Done.